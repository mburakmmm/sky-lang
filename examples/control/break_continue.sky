# Break and Continue test

function main
  print("=== Break Test ===")
  let i = 0
  while i < 10
    if i == 5
      break
    end
    print(i)
    i = i + 1
  end

  print("=== Continue Test ===")
  let j = 0
  while j < 10
    j = j + 1
    if j == 3
      continue
    end
    if j == 7
      continue
    end
    print(j)
  end

  print("=== Done ===")
end

