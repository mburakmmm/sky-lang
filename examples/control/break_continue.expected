=== Break Test ===
0
1
2
3
4
=== Continue Test ===
1
2
4
5
6
8
9
10
=== Done ===

